### ISSUE TITLE
Nvda is not able to open the link on a page

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#6818]
Testing on a page of a site in Portuguese - Brazil 
Http://3remetodista.org.br/

When pressing enter on some links nvda has a very strange behavior.
When you press enter on the link, the new content appears on the screen, but nvda does not read.
The only way to read the new page is to right-click and open the option to open a new page.
To reproduce the problem:
Open the site
Http://3remetodista.org.br/
Walk with arrows down until the excerpt:

Programe-se com o calendário regional e distrital de atividades da Igreja Metodista para o ano de 2017.

Press one more arrow below to get the link

visualizar

Press enter to open the link.
Nvda reads another section of the page, and does not open the link, follow the nvda log with the press of enter on the link:

Speaking [u'visitado  linque', u'Visualizar']
IO - inputCore.InputManager.executeGesture (13:13:15):
Input: kb(laptop):enter
DEBUGWARNING - virtualBuffers.gecko_ia2.Gecko_ia2._activateNVDAObject (13:13:15):
doAction failed
DEBUGWARNING - virtualBuffers.gecko_ia2.Gecko_ia2._activateNVDAObject (13:13:15):
Clicking with mouse
IO - speech.speak (13:13:15):
Speaking [u'clic\xe1vel', u' ', u'clic\xe1vel', u' ', u'Seja bem-vindo (a) ao novo site da Igreja Metodista\nna Terceira Regi\xe3o Eclesi\xe1stica,\xa0um site com uma nova cara,\ncom um aspecto\xa0moderno, inovador, simples e objetivo.', u'Nele voc\xea poder\xe1 acompanhar tudo o que se refere a 3\xaa regi\xe3o\nbem como dos distritos que a comp\xf5em.', u'Estamos evoluindo nossos meios de comunica\xe7\xe3o para expandir\no evangelho de Cristo Jesus,\xa0contamos com a colabora\xe7\xe3o\nde todos (as) para o crescimento desta ferramenta.\nDeus aben\xe7oe!', u'clic\xe1vel', u'Pr\xf3ximo', u'linque', u'3re_fonte_branca.png']
IO - inputCore.InputManager.executeGesture (13:13:19):
Input: kb(laptop):NVDA+f1
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IA2Attributes (13:13:19):
IAccessibleObject.attributes COMError (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (13:13:20):
Developer info for navigator object:
name: None
role: ROLE_UNKNOWN
states: 
isFocusable: False
hasFocus: False
Python object: <NVDAObjects.IAccessible.mozilla.Mozilla object at 0x0B1A7270>
Python class mro: (<class 'NVDAObjects.IAccessible.mozilla.Mozilla'>, <class 'NVDAObjects.IAccessible.ia2Web.Ia2Web'>, <class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: None
value: None
appModule: <'firefox' (appName u'firefox', process ID 14912) at address 516f930>
appModule.productName: u'Firefox'
appModule.productVersion: u'51.0.1'
TextInfo: <class 'NVDAObjects.IAccessible.IA2TextTextInfo'>
windowHandle: 1640372L
windowClassName: u'MozillaContentWindowClass'
windowControlID: 0
windowStyle: 1342177280
windowThreadID: 14932
windowText: u'NetscapeDispatchWnd'
displayText: exception: 'NoneType' object is not iterable
IAccessibleObject: <POINTER(IAccessible2) ptr=0xda4cd2c at b0c4ad0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=1640372, objectID=-4, childID=-701
IAccessible accName: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accRole: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accState: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accDescription: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accValue: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible2 windowHandle: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible2 uniqueID: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible2 role: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible2 states: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible2 attributes: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))

The only way to open the link is by right-clicking and clicking open on a new page.
The problem occurs in some other site links like:

Em 24 de maio de 1738 John Wesley teve sua vida radicalmente mudada através de uma experiência religiosa. Sentindo-se amado por Deus e, desafiado a criar um movimento para renovar a Igreja, ajudou as pessoas a conhecerem o amor de Deus e a trabalharem por um mundo melhor. Assim, no dia 20 de maio deste ano, vamos celebrar este marco para o povo metodista. Guarde esta data! Aguarde maiores informações.
"link" Saiba mais

SEMPRE TERÁ UMA PERTO DE VOCÊ!
"link" LOCALIZAR

Tested using nvda next 13840 and firefox 51.01, chrome 56 has the same problem.

**Comment by Brian1Gaff** [comment]
Do you by any chance have an ad blocker on the system, as when I had a look 
I get all sorts of errors on the page like the server cannot be found and 
content unavailable, please try later etc, before I get far enough down to 
get to the link you mention.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by fernando-jose-silva** [comment]
I believe that site is still under construction. Plus the links I said work, when you use the option to open in a new page the links that I said function. I do not even have an add-on in Firefox.

Enviado do meu iPhone

> Em 28 de jan de 2017, às 15:44, Brian1Gaff <notifications@github.com> escreveu:
> 
> Do you by any chance have an ad blocker on the system, as when I had a look 
> I get all sorts of errors on the page like the server cannot be found and 
> content unavailable, please try later etc, before I get far enough down to 
> get to the link you mention.
> Brian

**Comment by feerrenrut** [comment]
I was able to use this link with NVDA Version: `next-13821,09325856`, Chrome `55.0.2883.87 m (64-bit)`, and `Firefox 50.1.0`
The link opens a PDF in a new tab.

**Comment by fernando-jose-silva** [comment]
I ran new tests with nvda 13840 on a machine with windows 7 and the problem did not occur.
Already with the machine with windows 10 the problem occurred againwindows 10 build 14393.
I did tests with nvda 13849 and 13840.
Firefox 50.1 and chrome 56.
I also tested disabled add-ons.
Is there a problem with nvda on windows 10?
Last question, windows and firefox and chrome in 64bit.

**Comment by feerrenrut** [comment]
I should have mentioned that I tested on Windows 10 build 14393.

**Comment by fernando-jose-silva** [comment]
Thank you very much.
I asked the website developers to look at the code to see if accessibility policies were adhering to.
I did not give a recommendation because I could not do this, my training does not include programming.
But the fact is that the developers of the site made changes, where now I can open the links using nvda and firefox.
I would not know what changed on the site.
Now my suggestion for improvements is only for the so-called 6817.

**Comment by feerrenrut** [comment]
In that case I will close this issue, please re-open or let me know if this issue is still relevant.
